avm/res/batch/batch-account/tests/e2e/encr/main.test.bicep

```diff
@@ -0,0 +1,86 @@
+targetScope = 'subscription'
+
+metadata name = 'Using Customer-Managed-Keys with User-Assigned identity'
+metadata description = 'This instance deploys the module using Customer-Managed-Keys using a User-Assigned Identity to access the Customer-Managed-Key secret.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-batch.batchaccounts-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
```

Comment:
It looks like we have Azure Batch accounts per region per subscription default limit set to 3. Should we change the deployment location for one of the tests?

Ref: https://learn.microsoft.com/en-us/azure/batch/batch-quota-limit#resource-quotas

Reply:
Instead of implement a workaround for one resource alone I'm wondering if we should invest some time to implement a CI-powered location rotation. What do you thing @eriqua?

Reply:
Updated & retested @eriqua